operator/src/main/java/oracle/kubernetes/operator/DomainProcessorImpl.java

```diff
@@ -784,12 +787,15 @@ public void onThrowable(Packet packet, Throwable throwable) {
 
   Step createDomainUpPlan(DomainPresenceInfo info) {
     Step managedServerStrategy =
-        bringManagedServersUp(DomainStatusUpdater.createEndProgressingStep(new TailStep()));
+        Step.chain(
+            DomainStatusUpdater.createProgressingStep(MANAGED_SERVERS_STARTING_PROGRESS_REASON, true, null),
```

Comment:
Is there some reason not to create this progressing state in the bringManagedServersUp method?

Reply:
There is no particular reason. I will make the change to create progressing state in bringManagedServersUp like this. Thanks.

  private static Step bringManagedServersUp(Step next) {
    return DomainStatusUpdater.createProgressingStep(MANAGED_SERVERS_STARTING_PROGRESS_REASON, true,
            new ManagedServersUpStep(next));
  }